### DISCUSSION TITLE
feat(MultiSelect): expand/collapse dropdown on focus in-out

### DISCUSSION THREAD

**Discussion by eramosr16** [dotnetcore/BootstrapBlazor#3770] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Cycling over select components with Tab don't show the selection options by default.

### Expected Behavior

When component get focus should display the option values.

**Comment by bb-auto** [comment]
@eramosr16 Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Comment by ArgoZhang** [comment]
@eramosr16 The drop-down box cannot be expanded by clicking. When the tab key is pressed. please sync the latest version before pushing pr make sure your commit is up to date

**Comment by eramosr16** [comment]
@ArgoZhang the dropdown display the options values when you click on it, but when the component get the focus by cycling through the form using Tab key this doesn't happen, the pr created should address this scenario.

**Comment by ArgoZhang** [comment]
@eramosr16 please fix the bug. 

repro steps

1. press `Tab` to trigger `focusin`
2. click dropdown item then collapse the dropdown-menu

### expected
select value is `SelectedItem`

### actual
the first dropdown-item
